content/docs/nextjs/github-public-repo.md

```diff
@@ -3,46 +3,82 @@ title: Open Authoring with GitHub
 
-## Requirements:
-
-- You must be using [Next.js v.9.3](https://nextjs.org/blog/next-9-3) or later.
-- If your site is static, is must be hosted on a service that supports Next.js [Preview Mode](https://nextjs.org/docs/advanced-features/preview-mode) (E.g [Zeit's Now](https://zeit.co/home)).
-- Your editable content must be stored in GitHub.
-- Your repository containing your content **must be public**.
+The [Preview Mode](https://nextjs.org/docs/advanced-features/preview-mode) offered by Next.js allows us to load a separate set of data [depending on the "edit" (or "preview") mode](/blog/introducing-visual-open-authoring#using-nextjs-to-enable-edit-mode). With the help of the GitHub API, we can allow anyone to fork your site, make changes, and create a pull request from the Tina UI. Instead of writing locally to the filesystem via Git, the Tina GitHub helpers source data and commit to an "Editing Repository".
 
 ## Summary
 
-1. [Install the required packages](/docs/nextjs/github-public-repo#installation)
-2. [Configure the custom _\_app_ file](/docs/nextjs/github-public-repo#configure-the-custom-app-file)
-3. [Setup API functions using hygen scripts](/docs/nextjs/github-public-repo#api-functions)
-4. [Add an Auth Redirect page component](/docs/nextjs/github-public-repo#auth-redirects)
-5. [Create the GitHub OAuth app](/docs/nextjs/github-public-repo#setup-the-github-oauth-app)
-6. [Load content from GitHub](/docs/nextjs/github-public-repo#loading-content-from-github) — using `getStaticProps` and [Preview Mode](https://nextjs.org/docs/advanced-features/preview-mode)
-7. [Create a Tina Form that sources content from GitHub](/docs/nextjs/github-public-repo#using-github-forms)
+1. [Setup _create-next-app_ in a new repository](/docs/nextjs/github-public-repo#using-create-next-app)
+2. [Install the required packages](/docs/nextjs/github-public-repo#install-tina-github-packages)
+3. [Configure the custom _\_app_ file](/docs/nextjs/github-public-repo#configure-the-custom-_-file)
+4. [Setup API functions using hygen scripts](/docs/nextjs/github-public-repo#adding-api-functions)
+5. [Add an Auth Redirect page component](/docs/nextjs/github-public-repo#create-an-auth-redirect-page)
+6. [Create the GitHub OAuth app](/docs/nextjs/github-public-repo#setup-the-github-oauth-app)
+7. [Load content from GitHub](/docs/nextjs/github-public-repo#loading-content-from-github) — using `getStaticProps` and [Preview Mode](https://nextjs.org/docs/advanced-features/preview-mode)
+8. [Create a Tina Form that sources content from GitHub](/docs/nextjs/github-public-repo#using-github-forms)
+
+## Using _create-next-app_
 
-## Installation
+Let's set up a [fresh install](https://nextjs.org/docs#setup) of _create-next-app_ and create a [new repository](https://github.com/new) in GitHub. It is important this this repository is _public_, otherwise the GitHub Helpers won't know how to find your content.
```

Comment:
```suggestion
Let's set up a [fresh install](https://nextjs.org/docs#setup) of _create-next-app_ and create a [new repository](https://github.com/new) in GitHub. It is important this repository is _public_, otherwise the GitHub Helpers won't know how to find your content.
```